gc/base/omrmmprivate.hdf

```diff
@@ -984,6 +984,32 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<data type="void *" name="concurrentStats" description="internal statistics for concurrent phase" />
 	</event>
 
+	<event>
+		<name>J9HOOK_MM_PRIVATE_CONCURRENT_MARK_PHASE_END</name>
+		<description>
+			Triggered when global concurrent phase work ends.
+			NOTE:  Only expected to be used by verbose GC output.
+		</description>
+		<struct>MM_ConcurrentMarkPhaseEndEvent</struct>
+		<data type="struct OMR_VMThread*" name="currentThread" description="current thread" />
+		<data type="uint64_t" name="timestamp" description="time of event" />
+		<data type="uintptr_t" name="eventid" description="unique identifier for event" />
+		<data type="void *" name="concurrentStats" description="internal statistics for concurrent phase" />
+	</event>
+
+	<event>
+		<name>J9HOOK_MM_PRIVATE_CONCURRENT_SCAVENGE_PHASE_END</name>
+		<description>
+			Triggered when scavenge concurrent phase work ends.
```

Comment:
With trigger name change please change description to mention "local" rather than "scavenge"